src/wp-includes/rest-api/endpoints/class-wp-rest-posts-controller.php

```diff
@@ -2917,6 +2918,16 @@ public function get_collection_params() {
 			);
 		}
 
+		$query_params['search_columns'] = array(
+			'default'           => array(),
```

Comment:
There is a small indentation issue in this array :)

Reply:
Fixed, thanks for catching that.